app/javascript/packs/fetch/requester.js

```diff
@@ -24,5 +41,10 @@ async function doFetchRequest(route, method, body) {
       'Content-Type': 'application/json'
     },
     credentials: 'same-origin'
-  }).then(resp => resp.json());
+  }).then(function success(response) {
```

Comment:
Whoops, I'll fix that! I think I originally named it success because I was trying to understand when the promise was resolving, and I thought the errors would fall into the catch case.